classes/PodsAPI.php

```diff
@@ -8331,220 +8331,13 @@ public function process_form ( $params, $obj = null, $fields = null, $thank_you
 	 * Get current language information from Multilingual plugins
 	 *
 	 * @since 2.6.6
+	 * @since 2.7 Logic moved to PodsI18n
 	 *
 	 * @return array
 	 */
 	public static function get_current_language() {
```

Comment:
Do we need to leave a proxy around in `PodsAPI`?  I don't see any references to `PodsAPI::get_current_language()` outside of itself so seems like this should just be removed and all calls made directly into `PodsI18n`